operator/pkg/central/extensions/reconcile_pvc.go

```diff
@@ -232,7 +232,7 @@ func (r *reconcilePVCExtensionRun) handleCreate(claimName string, pvcConfig *pla
 
 		Spec: corev1.PersistentVolumeClaimSpec{
 			AccessModes: []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce},
-			Resources: corev1.ResourceRequirements{
+			Resources: corev1.VolumeResourceRequirements{
```

Comment:
Can you clarify why the code change is related to the version bump?

Reply:
It was changed in the upstream to prevent some erros 
- https://github.com/kubernetes/api/commit/8b14183ca44824850577019d995e6f9e811825a2